components/retro-go/rg_display.c

```diff
@@ -157,7 +161,7 @@ static void spi_init(void)
     ret = spi_bus_add_device(RG_GPIO_LCD_HOST, &devcfg, &spi_dev);
     RG_ASSERT(ret == ESP_OK, "spi_bus_add_device failed.");
 
-    xTaskCreatePinnedToCore(&spi_task, "spi_task", 1024, NULL, RG_TASK_PRIORITY - 1, NULL, 1);
```

Comment:
This should be undone, I think you didn't pull changes I've made before pushing.